docs/configuring-crash-stats.rst

```diff
@@ -0,0 +1,310 @@
+      socorro.external.postgresql.crashstorage.PostgreSQLCrashStorage'
+
+
+Start services
+--------------
+
+Both the Django socorro-webapp and the socorro-middleware REST service
+must be running::
+
+    sudo systemctl enable socorro-middleware socorro-webapp
+
+Configure Nginx
+---------------
+
+Both socorro-webapp and socorro-middleware should be fronted by a
+webserver like Nginx.
+
+You can find a working configs in
+/etc/nginx/conf.d/socorro-{webapp,middleware}.conf.sample
+
+You should change server_name in socorro-webapp.conf at minimum, the default is
+"crash-stats".
+
+You can leave the default "socorro-middleware" in socorro-middleware.conf
+
+Copy these .sample files to .conf and restart Nginx to activate::
+
+  sudo systemctl restart nginx
+
+Cron jobs
+---------
+
+Socorro uses a crontab manager called Crontabber. This needs
+to be in /etc/cron.d/socorro on a single host (generally referred to
+as the "admin host")::
+
+    */5 * * * * socorro /data/socorro/application/scripts/crons/crontabber.sh
+
+Set up crash-stats web site
+---------------------------
+
+Socorro produces graphs and reports, most are updated once per day.
+
+You must enter information about your releases into Socorro in order
+for this to work, and this information must match the incoming crashes.
+
+Becoming a superuser
+--------------------
+
+If you're starting a fresh new Socorro instance without any users at
+all, you need to bootstrap at least one superuser so the paragraph
+above starts to make sense. To do that, you first need to **sign in at
+least once** using the email address you want to identify as a
+superuser. Once you've done that, run the following command::
+
+    cd /data/socorro
+    export SECRET_KEY="..."
+    ./socorro-virtualenv/bin/python webapp-django/manage.py makesuperuser theemail@address.com
+
+Now the user with this email address should see a link to "Admin" in
+the footer.
+
+From this point on, you no longer need the command line to add other
+superusers - you can do this from http://crash-stats/admin/users/
+
+Adding new products and releases
+--------------------------------
+
+Each product you wish to have reports on must be added via the Socorro
+admin UI:
+
+http://crash-stats/admin/products/
+
+All products must have one or more releases:
+
+http://crash-stats/admin/releases/
+
+The new releases should be "featured" so they are
+used as defaults and show up in all reports:
+
+http://crash-stats/admin/featured-versions/
+
+Make sure to restart memcached so you see your changes right away:
+::
+  sudo systemctl restart memcached
+
+Now go to the front page for your application. For example, if your application
+was named "KillerApp" then it will appear at:
+
+http://crash-stats/home/products/KillerApp
+
+You should also change the DEFAULT_PRODUCT in local.py (/etc/socorro/local.py
+in a packaged install, ./webapp-django/crashstats/settings/local.py otherwise).
+
+Active Daily Install (ADI)
+--------------------------
+
```

Comment:
Yes this whole document is, that's why we're warning people away from PG+Crash-Stats right? :)